named-tunnel-k8s/README.md

```diff
@@ -1,66 +1,63 @@
-This is an example of using a [named tunnel (beta)](https://community.cloudflare.com/t/argo-tunnel-named-tunnel-beta/202445) in kubernetes. 
+This is an example of using a [Cloudflare Tunnel](https://developers.cloudflare.com/cloudflare-one/connections/connect-apps) (formerly Argo Tunnel) in Kubernetes. The architecture we suggest is running your app in a Kubernetes [Service](https://kubernetes.io/docs/concepts/services-networking/service/), and then running cloudflared in a separate [Deployment](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/). cloudflared will proxy internet traffic into your app using its service address.
```

Comment:
We should also mention that if they have an ingress controller, such as envoy, they can put cloudflared in front of the ingress controller.